playground/css/main.js

```diff
@@ -93,6 +97,10 @@ Promise.all(
 })
 
 // globEager
+// warning please use import.meta.glob('./glob-import/*.css', { query: '?inline' })
+// IF Not Use Default Should No Warning console https://github.com/vitejs/vite/issues/12001#issuecomment-1612411026
```

Comment:
```suggestion
// No warning to use `{ query: '?inline' }` is logged because `default` isn't accessed
// https://github.com/vitejs/vite/issues/12001#issuecomment-1612411026
```